app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReviewsController < ApplicationController
+  def new
+    @review = Review.new
+  end
+
+  def create
+    @booking = Booking.find(params[:booking_id])
+    @review = Review.new(review_params)
+    @review.booking = @booking
+    if @review.save
+      redirect_to booking_path(@booking)
+    else
+      render "booking_reviews"
+    end
+  end
+
```

Comment:
is private not needed in this line?

Reply:
Thank you - have added private in line 19